channels/views_test.py

```diff
@@ -691,10 +721,14 @@ def test_add_contributor(client, use_betamax, praw_settings, staff_jwt_header):
     """
     Adds a contributor to a channel
     """
-    client.force_login(UserFactory.create(username='fooadmin'))
-    contributor = UserFactory.create(username='othercontributor')
+    admin = ProfileFactory.create()
+    admin.user.username = 'fooadmin'
```

Comment:
This is missing a `.save()`, but you can also change this to `ProfileFactory.create(user__username='fooadmin')`